playbooks/templates/.env.j2

```diff
@@ -138,63 +179,85 @@ SKYNET_ACCOUNTS_LOG_LEVEL=info
 #   the default value set by skynet-webportal/docker-compose.accounts.yml if
 #   this variable is empty or undefined.
 # - 'false': All users incl. anonymous users can access the portal (access to portal is public)
+# REQUIRED: manually defined by user, or default is set in docker-compose file
 ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access }}
 
 {% endif %}
 # Accounts email
+# REQUIRED: manually defined by user
 ACCOUNTS_EMAIL_URI={{ webportal_common_config.accounts_email_uri }}
 
 # Domain to which your cookies will be issued
+#
+# REQUIRED: See top of file 
```

Comment:
do we want to note that vars like this are required only if accounts module (or some other module) is enabled?

Reply:
this is already grouped like that :-)

This is under the `(optional) Accounts` section.